src/PowerShellEditorServices/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -32,8 +32,7 @@ public class PesterDocumentSymbolProvider : FeatureProviderBase, IDocumentSymbol
 
             return commandAsts.OfType<CommandAst>()
                               .Where(IsPesterCommand)
-                              .Select(ast => ConvertPesterAstToSymbolReference(scriptFile, ast))
-                              .Where(pesterSymbol => pesterSymbol?.TestName != null);
+                              .Select(ast => ConvertPesterAstToSymbolReference(scriptFile, ast));
```

Comment:
Do you know if the callers deal with nulls in the result?

Reply:
Yeah, that is the code change I made in the VSCode extension - to handle a null TestName.